lib/ErrorSerializer.js

```diff
@@ -0,0 +1,20 @@
+var serializeError = require('serialize-error');
```

Comment:
Personally I don't have any problems with adding dependencies but if this doesn't feel right for you, we can inline the content of that package into the ErrorSerializer. But this means that we have to add also their tests to avoid coverage regressions?